contributors/devel/development.md

```diff
@@ -384,12 +389,15 @@ Upon merge (by either you or your reviewer), all commits left on the review
 branch should represent meaningful milestones or units of work.  Use commits to
 add clarity to the development and review process.
 
-Before merging a PR, squash any _fix review feedback_, _typo_, and _rebased_
-sorts of commits.
+Before merging a PR, squash any _fix review feedback_, _typo_, _merged_, and
```

Comment:
I know this is something contributors can just google for, but if we're going to explain commands for getting the branch updated, can we also explain how to squash.